scripts/AudioManager.js

```diff
@@ -83,16 +83,41 @@ function AudioManager(addEventListener, isTownTune) {
 					audio.volume -= step;
 				} else {
 					clearInterval(fade);
-					audio.pause();
+					playPause(false);
 					audio.volume = oldVolume;
 					if (callback) callback();
 				}
 			}, 100);
 		}
 	}
+	
+	function playPause(play) {
```

Comment:
I think it would be a good idea to remove it then so future developers aren't tempted to do something similar.